depthai_nodes/parsing_neural_network.py

```diff
@@ -38,15 +40,52 @@ def __init__(self, *args, **kwargs) -> None:
         self._parsers: dict[int, BaseParser] = {}
 
     def build(
-        self, input: dai.Node.Output, nnArchive: dai.NNArchive
+        self,
```

Comment:
Should we rather `overload` the `build` function here - use `overload` decorator from `typing`? So one version where it accepts the NNModelDescription and one where it accepts NNArchive?

Reply:
Added in [7662ce2](https://github.com/luxonis/depthai-nodes/pull/105/commits/7662ce2f3b7f42e74b955b099884aa70aff761b2).